src/react-components/avatar-selector.js

```diff
@@ -157,14 +138,10 @@ class AvatarSelector extends Component {
         <a-scene vr-mode-ui="enabled: false" ref={sce => (this.scene = sce)}>
           <a-assets>{avatarAssets}</a-assets>
 
-          <a-entity rotation={`0 ${initialRotation} 0`}>
-            <a-animation
-              ref={anm => (this.animation = anm)}
-              attribute="rotation"
-              dur="2000"
-              easing="ease-out"
-              to={`0 ${toRotation} 0`}
-            />
+          <a-entity
+            rotation={`0 ${initialRotation} 0`}
+            animation={`property: rotation; dur: 2000; easing: easeOutQuad; to: 0 ${toRotation} 0;`}
```

Comment:
Surprised this ends up working without needing to set the matrix dirty flag in the animation component...